src/main/java/com/github/bordertech/config/DefaultConfiguration.java

```diff
@@ -1530,7 +1353,7 @@ protected void addOrModifyProperty(final String name, final String value) {
 
 		recordMessage("modifyProperties() - Adding property '" + name + "' with the value '" + value + "'.");
 
-		runtimeProperties.setProperty(name, value);
+		new IncludeProperties("Runtime: added at runtime").put(name, value);
```

Comment:
Do we need a new instance each time? Or can we use the one `runtimeProperties` instance?

Reply:
Reinstated the Runtime instance